File: lat2_java/hardware.java
class hardware extends product{
    private String brand;
    private String model;
    
    public hardware(){
    }
    
    public hardware(String brand, String model){
        this.brand = brand;
        this.model = model;
    }

    public String getBrand(){
        return brand;
    }

    public void setBrand(String brand){
        this.brand = brand;
    }

    public String getModel(){
        return model;
    }

    public void setModel(String model){
        this.model = model;
    }

    public void printHardware(){
        System.out.print("Brand : ");
        System.out.println(getBrand());
        System.out.print("Model : ");
        System.out.println(getModel());
    }
}

File: lat2_python/hardware.py
class hardware(product):

    __brand = ""
    __model = ""

    def __init__(self, brand = "", model = ""):
        self.__brand = brand
        self.__model = model

    def setbrand(self, brand):
        self.__brand = brand

    def setmodel(self, model):
        self.__model = model

    def getbrand(self):
        return self.__brand

    def getmodel(self):
        return self.__model

    def printHardware(self):
        print("Brand : ", getbrand())
        print("Model : ", getmodel())
        